newIDE/app/src/ObjectsList/index.js

```diff
@@ -142,6 +142,8 @@ type StateType = {|
   renamedObjectWithScope: ?ObjectWithContext,
   variablesEditedObject: any,
   searchText: string,
+  makeNewInstanceOnAdd: boolean,
+  addInstancePosition: Array<number>,
```

Comment:
I think this can be done with only addInstancePosition - then detect that makeNewInstanceOnAdd is false if the array is empty